etna/analysis/plotters.py

```diff
@@ -1336,27 +1395,22 @@ def plot_holidays(
 
         * if str, then this is code of the country in `holidays <https://pypi.org/project/holidays/>`_ library;
 
-        * | if DataFrame, then dataframe with holidays is expected to have timestamp index with holiday names columns.
-          | In a holiday column values 0 represent absence of holiday in that timestamp, 1 represent the presence.
+        * if DataFrame, then dataframe is expected to be in prophet`s holiday format;
```

Comment:
I think we should add here some info about as_is logic.

Reply:
Added on line 1409

Reply:
User starts reading documentation, during the reading about `holidays` param he fixed in his head the expected format, but few lines later there is a description of `as_is` format that discovers that `holidays` can be in the other format. 

I think that documentation of `holidays` should cover that its behavior depends on `as_is` parameter and explain it. I don't think that description of format should be spread across two parameters: `holidays` and `as_is`.